pkg/machine/applehv/machine.go

```diff
@@ -904,10 +901,10 @@ func (m *MacMachine) setupAPIForwarding(cmd []string) ([]string, string, machine
 		forwardUser = "root"
 	}
 
-	cmd = append(cmd, []string{"-forward-sock", socket.GetPath()}...)
-	cmd = append(cmd, []string{"-forward-dest", destSock}...)
-	cmd = append(cmd, []string{"-forward-user", forwardUser}...)
-	cmd = append(cmd, []string{"-forward-identity", m.IdentityPath}...)
+	cmd.AddForwardSock(socket.GetPath())
```

Comment:
i wonder ... should we follow a pattern here like:

cmd := gvproxy.Command().AddForwardSock(...).AddForwardDest(...)...

and then if we must afterwards:

cmd.foo = "bar"

Reply:
We can definitely do that if you don't mind that it blocks this PR

Reply:
I was giving this a go and, this is just me, but when we start chaining 2-3 functions the line gets really long. I think this is a little difficult to read when you have to go really far to the edge of the screen versus just reading straight down. Regardless, I can still implement it if it's what we want.